SeeThru_Feeds/Model/Scripts/ScriptBase.py

```diff
@@ -240,5 +250,10 @@ def SetInternalAlias(self, alias):
 
         Arguments:
             alias {String} -- The new internal alias
+
+        Returns:
```

Comment:
The docstring style is something I have not seen before. Is this confirming to a certain PEP standard and if yes, could you point me to it?

Reply:
This is the google docstring format, https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
This is the default docstring format used with the Python Docstring Generator on vs code and I found it to be the clearest of the options available